packages/tooltip/src/Tooltip.tsx

```diff
@@ -103,6 +108,11 @@ const minSize = notchWidth + 2 * borderRadius;
 const minHeightStyle = css`
   min-height: ${minSize}px;
 `;
+
+const transitionDelay = css`
+  transition-delay: ${transitionDuration.slowest}ms;
```

Comment:
Should this timing be customizable?

Reply:
Or at least, should the delay be optional?

Reply:
we're waiting until `infoSprinkle` comes out to make this optional